### PR TITLE
LLT-5040 Fix nat-lab issues around dockers

### PR DESCRIPTION
**Pull request by jjanowsk**
Bug no. 1: Bug in docker compose: https://github.com/docker/compose/issues/11510
This bug randomly causes missing network interfaces in started containers. We just updated docker compose version to the version which is not affected.

Bug no. 2: Bug in docker daemon: https://github.com/moby/moby/issues/47619
Because of this bug eth0 is not created before sysctls are being set so it is not possible to set any sysctls related to the specific interfaces.
We can create a workaround by not setting eth0. Instead we can set default so eth0 will have this value assigned later when created.

### :ballot_box_with_check: Definition of Done checklist
- [x] Commit history is clean ([requirements](../blob/main/docs/git_commit_messages_requirements.md))
- [x] README.md is updated
- [x] changelog.md is updated
- [x] Functionality is covered by unit or integration tests

### CODE REVIEW

**Review by arg0d — DISMISSED** [review]
Looks good.

File: Cargo.lock [hunk 1]
```diff
@@ -1320,18 +1320,18 @@ dependencies = [
 
 [[package]]
 name = "enum-map"
-version = "2.7.2"
+version = "2.7.3"
```
Comment (arg0d) [inline Cargo.lock:1323]: Is it necessary to update `Cargo.lock`?

**Review by matszczygiel — APPROVED** [review]
+1

File: Cargo.toml [hunk 2]
```diff
@@ -21,7 +21,7 @@ cfg-if = "1.0.0"
 ffi_helpers = "0.3.0"
 num_cpus = "1.15.0"
 wasm-bindgen = "=0.2.83" # Only for compatability with moose wasm version
-h2 = "=0.3.24" # RUSTSEC-2024-0003
+h2 = "=0.3.26" # RUSTSEC-2024-0332
```
Comment (tomaszklak) [inline Cargo.toml:24]: that is unrelated change, already being reviewed in #464

**Review by tomaszklak — CHANGES_REQUESTED** [review]
I think security related changes should have separate PRs

File: Cargo.toml (hunk 2 above)
Comment (jjanowsk) [reply Cargo.toml:24]: So you would now recommend everyone to wait for #464 to be merged and block any other PRs? I don't see a point. For me it should be whoever merges it first. It's in a separate commit with a description. I don't see why it must also be in a separate PR.

Comment (tomaszklak) [reply Cargo.toml:24]: I don't think that this is urgent enough to make a mess.

**Review by packgron — APPROVED** [review]
+1.0

File: Cargo.lock (hunk 1 above)
Comment (jjanowsk) [reply Cargo.lock]: It's outdated, but the answer is: yes

**Review by tomaszklak — APPROVED** [review]
+0.5
